### DISCUSSION TITLE
Lógica de Programação com Dart: Link do Notion

### DISCUSSION THREAD

**Discussion by clestonv** [balta-io/forum#396] [Tira Dúvidas] [answered]
Duranta a introdução do curso na aulta **Conceitos Básicos - Variáveis**, o Gabu comenta sobre um link do Notion mas não encontrei, essa doc existe mesmo ou não?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @clestonv, como vai? Tem algum conteúdo neste link que você precisa? Estou vendo se consigo disponibilizar, mas enquanto isso, pode mandar suas dúvidas aqui mesmo.

**Comment by ghost** [comment]
olá eu estou iniciando nesse mundo da tecnologia e da programação, e resolvi começar o curso de vocês de lógica da programação com Dart . Então gostaria de saber se com esse curso essa lógica da programação só se aplica ao Dart ou eu vou poder usar a base para programar em outras linguagens como C# , JAVA e outros.
